CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# How to contirbute
+
+## Repo structure
+
+The document table of contents are defined in the `toc.md` file and each of the sections show as different `.md` files from the repository:
+
+* Introduction
+* Contirbution to existing projects
+* Starting Open Source projects
+* References
+
+## Types of contirbutions
+
+There are many ways people can contribute to the content of this repo:
+
+* Propose new sections
+* Add new content to existing sections
+* Fix inaccuracies or inconsintencies
+* Add new references
+* Translate content into other languages
```

Comment:
I think we would want to reach a "first release" state before we ask for translations.

Reply:
Makes sense. Agree.

Reply:
OK! I've deleted that part in a new commit 👍